.circleci/config.yml

```diff
@@ -5,6 +5,19 @@ defaults: &defaults
   macos:
     xcode: '9.3.1'
```

Comment:
could we upgrade this to 10.1 or higher to get the right version of node and avoid having to use nvm? https://discuss.circleci.com/t/xcode-10-1-image-released/26350

Reply:
I've been hesitant to upgrade the version of XCode on these images without first investigating whether things will be fine for native modules (Electron 3.x runs on [macOS 10.9 or later](https://github.com/electron/electron/blob/024cb4afd9cebe9a1bf1976ba83c3cdce1a5afd7/docs/tutorial/support.md#macos), so that's our compatibility baseline we need to be mindful of).

Also, based on that link it looks like this takes us up to Node 11 (which is weird, given Node 12 is latest and Node 10 is latest LTS).

The recommendations in other support threads about "how do I enable a specific version of Node?" when I was looking into this all seem to coalesce around using NVM, so I'm not sure how unavoidable it is (short-term or longer-term) if we want to be able to control the version of Node used here without also upgrading XCode, like we can for the other providers.

Thoughts?

Reply:
maybe #7836 would work?